### PR TITLE
[WIP] nydus-image: fix unpack from specify blob dir

### PR DESCRIPTION
**Pull request by imeoer**
Fix the panic when user specify `--blob-dir` on unpack:

```
$ nydus-image unpack --bootstrap /path/to/bootstrap \
    --blob-dir /path/to/blobs \
    --output /path/to/oci.tar

Error: fail to unpack

Caused by:
    both blob path or blob backend config are not specified
```

## Relevant Issue (if applicable)
_If there are Issues related to this PullRequest, please list it._

## Details
_Please describe the details of PullRequest._

## Types of changes

_What types of changes does your PullRequest introduce? Put an `x` in all the boxes that apply:_

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation Update (if none of the other choices apply)

## Checklist

_Go over all the following points, and put an `x` in all the boxes that apply._

- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.

### CODE REVIEW

**Comment by adamqqqplay** [comment]
Really helpful fix, I thought it was a problem with my usage, so I could only use the --blob parameter instead of --blob-dir.

Thanks a lot!

File: src/bin/nydus-image/main.rs [hunk 1]
```diff
@@ -1172,7 +1172,13 @@ impl Command {
 
                 Some(Arc::new(local_fs))
             }
-            None => Self::get_backend(matches, "unpacker").ok(),
+            None => {
+                Some(BlobFactory::new_backend(
+                    config.backend.as_ref().unwrap(),
```
Comment (jiangliu) [inline src/bin/nydus-image/main.rs:1177]: Is it safe to unwrap() here? Or should we check and return error code instead?

**Comment by jiangliu** [comment]
Otherwise LGTM!

**Comment by imeoer** [comment]
Fixed in master.

**Pull request closed without merging**